client/state/data-layer/wpcom/comments/index.js

```diff
@@ -188,6 +191,68 @@ export const announceDeleteFailure = ( action ) => {
 	];
 };
 
+export const emptyComments = ( action ) => ( dispatch ) => {
+	const { siteId, status } = action;
+
+	dispatch(
+		http(
+			{
+				apiVersion: '1',
+				body: {
+					empty_status: status,
+				},
+				method: 'POST',
+				path: `/sites/${ siteId }/comments/delete`,
+			},
+			{
+				...action,
+			}
```

Comment:
Perhaps we could just pass `action` here? Is there a good reason why we create a new object?

```suggestion
			action
```